### ISSUE TITLE
Dataclass support doesn't fully resolve ClassVar annotation

### ISSUE THREAD

**Issue by abelcheung** [microsoft/pylance-release#876]
## Environment data

-   Language Server version: 2021.1.3-pre.1, 2021.1.2, and possibly earlier versions affected as well
-   OS and version: Windows 7
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.2

## Issue

Pylance complains with following sample code, with type checking mode set to `basic` or `strict`:

```python
import typing
from dataclasses import dataclass

@dataclass
class FooBar:

    attr1: int = 1
    attr2: typing.ClassVar[str]
```

Screenshot here:

![Type checking warning](https://user-images.githubusercontent.com/83110/105642704-289c7100-5e83-11eb-9fe6-8dfee9eeb371.png)

The warning persists when `ClassVar` is imported under any alias, like `ClassV`, `typing.ClassVar`, `whatever.ClassVar`. The warning only goes away if I use the _exact_ word `ClassVar` in annotation. That is replacing first and last line in above sample code with:

```python
from typing import ClassVar
    attr2: ClassVar[str]
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.1.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202113-27-january-2021
